boot/core/src/boot/task/built_in.clj

```diff
@@ -200,7 +200,14 @@
   - :type, :file, and :theme.
 
   The visual notification function will receive a map with four keys
-  - :title, :uid, :icon, and :message."
+  - :title, :uid, :icon, and :message.
+
+  The `notify` task always attempts to catch any exceptions that are
+  thrown so as not to cause your build to fail. If no notifications
+  are happening even if you specify `audible` or `visual`, you can see
+  any exceptions that are being caught by changing
+  `boot.util/*verbosity*` (at the command-line use: `boot -v` or `boot
+  -vv`)."
```

Comment:
I think at default verbosity level the exception is still actually printed with `print-ex`.  I'm still not super happy with what this says, but I like logging a warning.

Maybe I should take this section out of the help and just log a warning with just: "an exception was thrown while trying to notify, use `boot -v` to see more detail"?